packages/xen-api/src/index.js

```diff
@@ -746,7 +746,10 @@ export class Xapi extends EventEmitter {
     this._sessionId = await pRetry(
       () =>
         this._interruptOnDisconnect(
-          this._call('session.login_with_password', params)
+          this._call('session.login_with_password', params).catch(error => {
+            error.call.params = '* obfuscated *'
+            throw error
+          })
```

Comment:
Wouldn't it be better to update `_call` to not add `params` if this method is called?

Reply:
By testing specifically the method name?

Reply:
Yes.